tracee-ebpf/pkg/stats/stats.go

```diff
@@ -0,0 +1,9 @@
+package stats
+
+type EbpfStats struct {
```

Comment:
I would merge `stats.go` in `metrics.go`, and I would place the package under `tracee-ebpf/metrics/` instead of `tracee-ebpf/pkg/metrics` (kind of what you did in tracee-rules)

Reply:
stats can't be merged into metrics since that causes circular dependencies between metrics (which needs tracee's object to register to prometheus) and tracee (which needs the stats protocol to output it). As such they need to be separate packages.